### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by ramereth**
- Lock nfs cookbook version to what we currently have in production
- Update ServerSpec tests to support CentOS 7
- Ensure we update Berks properly

### CODE REVIEW

File: test/integration/helpers/serverspec/shared_examples.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+shared_examples 'nfs' do
+  if os[:release].to_i < 7
+    describe service('nfs') do
+      it { should be_enabled }
+      it { should be_running }
+    end
+  else
+    describe service('nfs') do
+      it { should be_enabled }
+    end
```
Comment (doublej472) [inline test/integration/helpers/serverspec/shared_examples.rb]: No `it { should be_running }`?

Comment (ramereth) [reply test/integration/helpers/serverspec/shared_examples.rb]: On CentOS 7, systemd just runs and exits for the 'nfs' service so this fails. This was the primary reason I fixed the tests :)

Comment (ramereth) [reply test/integration/helpers/serverspec/shared_examples.rb]: I'll add a comment here so that it's clear.

Comment (doublej472) [reply test/integration/helpers/serverspec/shared_examples.rb]: Strange, it worked for me when I added it before, now it's failing when I tear down and start the VM...

**Found out why!** If you:
1) converge the recipe
2) run `systemctl status nfs` on the VM
3) run `kitchen verify default-centos-7`

The verify will work, somehow. I'm not sure how it magically changes the kitchen test, but this PR is correct.

**Comment by ramereth** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v0.2.2.  Have a nice day!

**Pull request merged**